bindings/python/iota_sdk/types/block/wrapper.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Dict
+from dacite import from_dict
+from iota_sdk.types.block.block import Block
+from iota_sdk.types.common import HexStr, json
+from iota_sdk.types.signature import Signature
+from iota_sdk.utils import Utils
+
+
+@json
+@dataclass
+class BlockWrapper:
+    """The two block types introduced in this TIP share some common fields which are contained in an outer wrapper.
```

Comment:
This code is not a TIP :trollface:

Reply:
exactly my thought when I c&p'ed it ... weird.

Reply:
I am open to suggestions. Also for the missing `BasicBlock` docs.

Reply:
I'm fine with it now